packages/slate-editor/src/components/ElementPlaceholder/ElementPlaceholder.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+import { ChickenNoSignalIllustration } from '#icons';
+
+import { VStack } from '../Stack';
+
+import { Container, Illustration, Title, Subtitle } from './';
+
+export default {
+    title: 'Components/ElementPlaceholder',
+};
+
+export function NoData() {
+    return (
+        <div style={{ width: 700 }}>
+            <Container onClick={() => console.log('Click')}>
+                <VStack spacing="2">
+                    <Illustration>
+                        <ChickenNoSignalIllustration />
+                    </Illustration>
+                    <VStack spacing="1">
+                        <Title>The selected Prezly Story is no longer available</Title>
+                        <Subtitle>
+                            Click to select another a bookmark of one of your stories
+                        </Subtitle>
+                    </VStack>
+                </VStack>
+            </Container>
```

Comment:
As always, I am not a fan of constructor components. I'd better see this a black-box component with input properties. There's no need to make it more complex than that at this stage. KISS.

Reply:
I think we're talking about complexity on different abstraction levels. Let me try to convince you by explaining my perspective on this :)

One of the base principles of software development is to hide complexity by abstracting it into something simple enough for people to use. This applies to UIs, APIs, and components as well.

By exposing all the internals of this placeholder component, you force the consumer to: 
- learn which building blocks the component has
- find an example of how they can be composed
- import 4 components: `Container`, `Title`, `Subtitle`, and `VStack`
- composed them in the correct order, making sure they do not produce an inconsistent look & feel

This is the complexity I'm talking about. 

In contrast, if it was a black-box component with properties exposing its inputs, it becomes trivial: you just provide all details and you get a consistent result with no knowledge of the internals. 

There are, of course, cases when it's impossible to avoid exposing this complexity (like UI libraries, or frameworks). But for me, the rule of thumb is to always avoid that unless it becomes necessary. In this particular case, there are no signs yet that someone will need to build a different combination of `Container`, `Title`, `Subtitle` and `VStack` structure. Thus I'd better go with a simpler-to-use implementation. Encapsulated black-box it is.

Thoughts?

Reply:
I see your point, but looks like we have different view on such components. I treat them as dumb renders like ui libraries. For examples here in story we have `Click to select another a bookmark of one of your stories` but for specific reason in the editor itself we do not have this line, so we would need to add render condition in the component what imo reads harder that JSX. Also according to designs we will have DND in such components and we will need add special components to handle such behavior and therefore make this component to accept more and more props. 

We are adding ui to the page even when we are not working on ui library, but face the same issues. I would like to follow common approaches on building ui components even if when we are not working on ui-kit because in general we are solving the same issues. 

Your points are true, but in my opinion they should be applied to components with business or complex logic that depends on other sources.

In my mind there should be components package that operates UI, and other parts that uses them to build UI and even do not create their own css, all css should be in the ui components. If there is a collision it means that there is a mistake in designs that they added smth extraordinary and it could be solved by our already created components, or there are really new changes and we need update our components. There should not be a case that we create special UI behavior like hover color, button background etc. for a page just because we have it in designs.

It looks like we have controversial opinion on the same base things and they can't be resolve in scope of this PR. I will change the code here to have one component, but would like to discuss approach of building ui components in future.